index.d.ts

```diff
@@ -5,6 +5,12 @@ Symbol event names can be used to avoid name collisions when your classes are ex
 */
 type EventName = string | symbol;
 
+/** The data provided as `eventData` when listening for `Emittery.listenerAdded` or `Emittery.listenerRemoved`. */
+interface EventListenerData {
```

Comment:
The name is too generic. And the interface should be exported.

Reply:
Fixed!